bitrise.yml

```diff
@@ -223,6 +224,23 @@ workflows:
             $HOME/.gradle/caches/*.lock
             ./.gradle/*.lock
             ./.gradle/*.bin
+  _create-github-release:
+    steps:
+    - script@1:
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -ex
+            
+            _CHANGELOG="${GIT_REPOSITORY_URL%.git}/blob/$BITRISE_GIT_TAG/inappmessaging/USERGUIDE.md#-changelog"
```

Comment:
will this only retrieve the changelog for the specific version?

Reply:
Yes, sample is in the [releases](https://github.com/rakutentech/android-inappmessaging/releases) page